tests/by-util/test_tail.rs

```diff
@@ -4479,3 +4479,264 @@ fn test_args_sleep_interval_when_illegal_argument_then_usage_error() {
             .code_is(1);
     }
 }
+
+#[test]
+fn test_gnu() {
```

Comment:
could you please split this test for what it is testing?
easier to debug if we are regressing

Reply:
i split them up into argument "type" (-c, +c, -l, +l, -b, -${num}, +${num}) and errors. removed the obvious tests that are already working on main